server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+public class ExtensionsOrchestratorTests extends OpenSearchTestCase {
+
+    public void testReadFromExtensionsYml() throws Exception {
+
+        ExtensionsSettings extensions = ExtensionsOrchestrator.readFromExtensionsYml("/config/extensions.yml");
+
+        assertNotNull(extensions);
```

Comment:
I would test that **all** fields within the extension.yml can be extracted and read correctly, rather than just two of the fields. But otherwise this looks good to me. :)